wavesrv/cmd/main-server.go

```diff
@@ -618,6 +619,15 @@ func HandleReadFile(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func HandleConfig(w http.ResponseWriter, r *http.Request) {
+	qvals := r.URL.Query()
+	filePath := qvals.Get("path")
+	log.Printf("URL query: %v %v\n", qvals, filePath)
+	fullPath := path.Join(scbase.GetWaveHomeDir(), "config", filePath)
```

Comment:
move this path definition to base.go (with the others).  "config" should be a const in there.